### ISSUE TITLE
min height and width

### ISSUE THREAD

**Issue by kingsleyh** [s0kil/mint-tauri-starter#1]
Hey

Do you know if it's possible to configure or set the min width and height so that the resizable window is not squashed tiny but stops at the min height and width.

**Comment by s0kil** [comment]
@kingsleyh Have you tried setting it for the `body` as in https://github.com/s0kil/mint-tauri-starter/blob/master/source/common.css ?

**Comment by kingsleyh** [comment]
yeah I tried putting:    min-height: 200px;  into that - but doesn't make any difference. I think something must need to be specified as min height at the window level e.g like in tauri.conf.json or something - but that only seems to allow resizable, height, width and not the min

**Comment by s0kil** [comment]
Turns out Tauri does not yet supoort such API.

**Comment by kingsleyh** [comment]
Do you think something like this repo could be created for Electron and Mint?

**Comment by s0kil** [comment]
Another option is to use [pywebview](https://pywebview.flowrl.com), it supports the [API](https://pywebview.flowrl.com/guide/api.html) you are looking for.
If you persist with Electron then [Nativefier](https://github.com/jiahaog/nativefier) is probably what you want, it also supports the [min width/height API](https://github.com/jiahaog/nativefier/blob/master/docs/api.md).

**Comment by kingsleyh** [comment]
I’ll try them out - I couldn’t get Mint to work in Electron earlier

**Comment by s0kil** [comment]
Could you please elaborate on the problem(s) you encountered?

**Comment by s0kil** [comment]
Here Is the Mint website as Electron app
![Screenshot from 2020-04-01 13-28-31](https://user-images.githubusercontent.com/24403355/78167567-b43ca100-741c-11ea-9ef7-657b858083fb.png)

**Comment by kingsleyh** [comment]
how did you make that work?

**Comment by kingsleyh** [comment]
I put the Mint app in the electron wrapper and just a white screen appeared

**Comment by s0kil** [comment]
- Serve the Mint output dist folder: `cd dist; python3 -m http.server `
- Start Nativefier on local server: `nativefier --name "Tako" "http://0.0.0.0:8000"`
- Nativefier will gather all assets required and package into Electron app.

![Screenshot from 2020-04-01 15-29-07](https://user-images.githubusercontent.com/24403355/78178703-105bf100-742e-11ea-8786-e5911e83e92c.png)

**Comment by s0kil** [comment]
@kingsleyh Tauri now adds new configuration options to set the [window width and height](https://github.com/s0kil/mint-tauri-starter/blob/master/src-tauri/tauri.conf.json#L57-L61).

Not exactly minimum window width and height, but maybe still useful.
